CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.8.0] - 2025-05-07
+
+### 🚀 Features
+
+- `pop build runtime` and `pop build --deterministic` (#510)
+- Hashing (#517)
+- Revive compatibility with feature flag (#500)
```

Comment:
```suggestion
- Contracts: Revive compatibility with feature flag (#500)
```

Reply:
As suggestion above, renamed to use (contracts) https://github.com/r0gue-io/pop-cli/pull/536/commits/6c0dc95143f37d5e86fdbe4248d1645b7db44f55